WebHostLib/stats.py

```diff
@@ -26,9 +26,11 @@ def get_db_data(known_games: typing.Set[str]) -> typing.Tuple[typing.Counter[str
     room: Room
     for room in select(room for room in Room if room.creation_time >= cutoff):
         for slot in room.seed.slots:
-            if slot.game in known_games:
-                total_games[slot.game] += 1
-                games_played[room.creation_time.date()][slot.game] += 1
+            current_game = slot.game
+            if slot.game not in known_games:
```

Comment:
Might as well just always use this
```suggestion
            if current_game not in known_games:
```